hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -0,0 +1,115 @@
+# Pi4 Bootloader Configuration
+
+You can display the current configuration using 
+```
+vcgencmd bootloader_config
+```
+
+To change these bootloader configuration items, you need to extract the configuration segment, make changes, re-insert it, then reprogram the EEPROM with the new bootloader. 
+
+```
+# Extract the configuration file
+cp /lib/firmware/raspberrypi/bootloader/beta/pieeprom-2019-09-10.bin pieeprom.bin
```

Comment:
@timg236 Should this be using a non-beta filename?
Is there any way to match up which file in `/lib/firmware/raspberrypi/bootloader/` matches up with the currently-flashed eeprom image?

Reply:
They all appear to be in the beta folder at the moment. I think we can leave as is for the moment, then adjust at a later date as necessary.